docs/README.md

```diff
@@ -20,7 +20,7 @@ ApostropheCMS is a full-featured, open source CMS built with Node.js that seeks
 - [**Getting started**](/guide/setting-up.md): Environment setup, installation, and an overview of the major version changes.
 - [**Guide**](/guide/introduction.md): Discussion of Apostrophe 3 features using lots of code examples.
 - [**Reference**](/reference/): Detailed reference sections for developers.
-- [**Coming from Apostrophe 2**](/guide/upgrading.md): A guide to feature and API changes from Apostrophe 2.
```

Comment:
Is this an existing page with a changing URL? Is there a plan for redirects & no loss of SEO?

Reply:
redirects are in the enhanceApps.js file.